src/distraction/SoundDistraction.java

```diff
@@ -0,0 +1,38 @@
+package distraction;
+
+import com.jme3.asset.AssetManager;
+import com.jme3.audio.AudioNode;
+import eu.opends.main.Simulator;
+
+/**
+ *
+ * @author Johnny
+ */
+public class SoundDistraction extends DistractionClass{
+    
+    private AudioNode soundTest;
+    private AssetManager manager;
+    
+    @Override
+    public void initialize(Simulator sim) {
```

Comment:
is there a reason the distraction classes use `initialize()` instead of a constructor? the latter would seem more natural to Java to me